public/app/core/components/AppChrome/History/HistoryWrapper.tsx

```diff
@@ -78,127 +82,205 @@ function HistoryEntryAppView({ entry, isSelected, onClick }: ItemProps) {
   const expandedLabel = isExpanded
     ? t('nav.history-wrapper.collapse', 'Collapse')
     : t('nav.history-wrapper.expand', 'Expand');
-
+  const entryIconLabel = isExpanded
+    ? t('nav.history-wrapper.icon-selected', 'Selected Entry')
+    : t('nav.history-wrapper.icon-unselected', 'Normal Entry');
   const selectedViewTime =
     isSelected &&
     entry.views.find((entry) => {
       return entry.url === window.location.href;
     })?.time;
 
   return (
-    <Stack direction="column" gap={1}>
-      <Stack>
-        {views.length > 0 ? (
-          <IconButton
-            name={isExpanded ? 'angle-down' : 'angle-right'}
-            onClick={() => setIsExpanded(!isExpanded)}
-            aria-label={expandedLabel}
-            className={styles.iconButton}
+    <span className={styles.marginBottom}>
```

Comment:
if this is a `span`, then all it's children should be inline elements as well (e.g. no `div`s). if all you're using this for is to set margin, the easier thing to do is probably remove it and make the first child a `Box` instead of `Stack`.

Reply:
Should I also change the `span` with `className={styles.timeline}` into a `div`?

Reply:
ok, will do the change then. Thanks for the explanation 🙂